frontend/src/components/Admin/CoffeeChats/CoffeeChats.tsx

```diff
@@ -4,11 +4,23 @@ import { createHash } from 'crypto';
 import { Button } from 'semantic-ui-react';
 import styles from './CoffeeChats.module.css';
 import CoffeeChatAPI from '../../../API/CoffeeChatAPI';
+import { useMembers } from '../../Common/FirestoreDataProvider';
+import CoffeeChatsBingoBoard from '../../Forms/CoffeeChatsForm/CoffeeChatsBingoBoard';
 
 const CoffeeChats: React.FC = () => {
   const [bingoBoard, setBingoBoard] = useState<string[][]>([[]]);
   const [isLoading, setIsLoading] = useState<boolean>(true);
   const [pendingChats, setPendingChats] = useState<CoffeeChat[]>([]);
+  const [specificBingoBoard, setSpecificBingoBoard] = useState<string[][]>([[]]);
```

Comment:
We can use the `bingoBoard` state, since specificBingoBoard will be exactly the same as bingoBoard and isn't mutated elsewhere.